about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script>
+      src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+      </script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="about.css">
+  <title>About Burning Circle Flight Academy</title>
+</head>
+<body>
+  <div class="top-section">
+    <div class="bar-container">
+    <ul class="bars">
+      <li>
+        <i class="fab fa-facebook-f"></i>
+      </li>
+      <li>
+        <i class="fab fa-twitter"></i>
+      </li>
+      <li>
+        <h4>English</h4>
+      </li>
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
+  </div>
+  <header>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="images/plane3.svg" alt="plane" class="plane"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
```

Comment:
- [ ] The `header` element should have similar spacing, borders, shadows, and paddings like the original design.

Reply:
But how do i Know the exact spacing for I cannot inspect the template. Like seriously this  code review is not a fair one to be honest with you

Reply:
done